test-data/unit/semanal-typeddict.test

```diff
@@ -47,7 +47,7 @@ MypyFile:1(
   ClassDef:2(
     A
     BaseType(
-      mypy_extensions._TypedDict[builtins.str])
+      mypy_extensions._TypedDict[Union[builtins.str]])
```

Comment:
Otherwise we will have problems like this ☝️

Reply:
Fixed